guide/14-deep-learning/FCN_MLModel_guide-final_draft.ipynb

```diff
@@ -0,0 +1,501 @@
+{
```

Comment:
<p>(1) Performs well with <s>comparitively less</s>  <strong>comparatively fewer</strong> data <strong>points</strong></p><p>(2) It has some universal parameters for <strong>fine-tuning</strong>, like <strong>the</strong> number of hidden layers</p><p>(3) Several <strong>scikit-learn</strong> models <strong>are</strong> available, <strong>and</strong> each <strong>needs</strong> its own set of unique parameters &amp; hyperparameters setting, followed by <strong>fine-tuning</strong></p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>